cmd/dockerd/daemon.go

```diff
@@ -479,6 +480,7 @@ func initRouter(opts routerOptions) {
 		swarmrouter.NewRouter(opts.cluster),
 		pluginrouter.NewRouter(opts.daemon.PluginManager()),
 		distributionrouter.NewRouter(opts.daemon.ImageService()),
+		grpcrouter.NewRouter(opts.buildBackend),
```

Comment:
Is there a list or a way the backends are registered? It would be nice if in the future any backend which implemented the GRPC registration interface was added.

Reply:
There is no central place other than this place. Would require more refactoring for only mild benefits, but I get that it'd be nicer.

Reply:
@dmcgowan updated.